ui/webui/test/check-review

```diff
@@ -74,11 +74,10 @@ class TestReview(anacondalib.VirtInstallMachineCase):
 
         i.open()
 
-        for _ in i.steps[:-1]:
-            if i.steps[i.get_current_page_id()] != i.steps.REVIEW:
-                i.next()
-            else:
-                i.begin_installation(should_fail=True, confirm_erase=False)
+        current_page = i.steps.WELCOME
```

Comment:
Can't this be `current_page = get_current_page()` , that seems cleaner and less hardcoding.